go.mod

```diff
@@ -64,5 +64,5 @@ replace (
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
 	// Patch for a race condition involving metadata-only
 	// informers until it can be resolved upstream:
-	sigs.k8s.io/controller-runtime v0.9.2 => github.com/benluddy/controller-runtime v0.9.3-0.20210720171926-9bcb99bd9bd3
+	sigs.k8s.io/controller-runtime v0.10.0 => github.com/benluddy/controller-runtime v0.9.3-0.20210720171926-9bcb99bd9bd3
```

Comment:
We'll need to drop this commit and rebase against master now that #2368 landed.

Reply:
Yep, just waiting on https://github.com/operator-framework/api/pull/203 to merge then I'll update the pin here and rebase onto master.